osrf/vrx
# Task 5 Update

## To test:
* Go to https://github.com/osrf/vrx/wiki/vrx_2022-task_tutorials
* Read general instructions
* Click link at the bottom for [Task 5: Acoustic Beacon Localization, Channel Navigation and Obstacle Avoidance](https://github.com/osrf/vrx/wiki/vrx_2022-task_tutorials)
* Run through the tutorial and make sure it works.
